platform/log/zephyr/src/mender-log.c

```diff
@@ -33,36 +33,47 @@ mender_log_init(void) {
     return MENDER_OK;
 }
 
+/* IMPORTANT: Please note that the default size of internal Zephyr logger buffer is 1024 bytes; if we need log buffer size 
+              to be more than that there is CONFIG_LOG_BUFFER_SIZE option allowing to extend the buffer size. */
+#define LOG_BUFFER_SIZE_BYTES (128)
+
 mender_err_t
 mender_log_print(uint8_t level, const char *filename, const char *function, int line, char *format, ...) {
 
+    (void)filename;
```

Comment:
Might as well use `ARG_UNUSED` in the argument list above instead.

Reply:
@vpodzime This is a miss-understanding between you and me, I figured it out only yesterday!

The Zephyr OS' `ARG_UNUSED` is not to be used in the argument list but here, while the one we defined in mender-utils.h is to be used in the argument list. Now, our custom one won't have effect for Zephyr OS builds and I also don't know how portable it is.

So yes, you were right [here by questioning me](https://github.com/mendersoftware/mender-mcu/pull/63#discussion_r1761000982), and I think we should stick to this style on platform code.

Reply:
Of course! We should have started with `MENDER_` prefix from the beginning :melting_face: 

I leave it to you and Vratislav to decide if we need both for our code. Do you care @larsewi to follow-up on this on an independent PR?

Reply:
> Of course! We should have started with `MENDER_` prefix from the beginning 🫠

I don't think that's necessary if we keep it in a header that's not included by any of the public API headers. But it doesn't hurt, I guess... Anyway, I prefer marking arguments as unused in the argument list rather than inside the functions.

Reply:
>  Do you care @larsewi to follow-up on this on an independent PR?

On it